branchExercise.txt

```diff
@@ -1,3 +1,6 @@
+line 1 branch fisher first commit
+line 2 branch fisher first commit
```

Comment:
Please make branch name uppercase in this comment

Reply:
that would cause a syntax error